docs/update_and_migration/from_4.6/update_from_4.6.md

```diff
@@ -138,8 +138,27 @@ merge with your custom settings if needed, and commit them to Git.
 
 ## v4.6.12
 
-If the new bundle `ibexa/core-search` has not been added by the recipies, enable it by adding the following line in `config/bundles.php`:
+If the new bundle `ibexa/core-search` has not been added by the recipes, enable it by adding the following line in `config/bundles.php`:
 
 ```php
     Ibexa\Bundle\CoreSearch\IbexaCoreSearchBundle::class => ['all' => true],
 ```
+
+## v4.6.13
+
+This release comes with a command to clean up the duplicated entries in the `ezcontentobject_attribute` table, caused by the issue described in [IBX-8562](https://issues.ibexa.co/browse/IBX-8562).
+
+If you're affected you can remove the duplicated entries by running the `ibexa:content:remove-duplicate-fields` command:
+``` bash
+php bin/console ibexa:content:remove-duplicate-fields
+```
+
+!!! caution
+
+    Remember about [**proper database backup**](backup.md) before running the command in the production environment.
+
+You can customize the behavior of the command with the following options:
+
+- `batch-size` or `b` - number of attributes affected per iteration. Default: 10000.
```

Comment:
```suggestion
- `batch-size` or `b` - number of attributes affected per iteration. Default value = 10000.
```
To make it consistent with existing docs (example: https://doc.ibexa.co/projects/userguide/en/latest/getting_started/dashboard/dashboard_block_reference/).